tests/unit/datarobot_drum/drum/test_drum.py

```diff
@@ -618,3 +618,35 @@ def test_runtime_params_not_present_as_env_var(
             assert file_mapping_param not in docker_cmd
             assert params_file_host_location not in docker_cmd
             assert params_file_docker_location not in docker_cmd
+
+
+class TestGetDockerTagNameFromDockerContextDir:
+    """Container for tests related to the method `_create_docker_tag_name`"""
+
+    @pytest.mark.parametrize(
+        "docker_context_dir, expected_tag",
+        [("/tmp/aaa", "tmp/aaa"), ("/tmp/aaa/bbb", "aaa/bbb"), ("/tmp/aa_a", "tmp/aa_a")],
+    )
+    def test_tag_from_valid_prefixes_and_suffixes(self, docker_context_dir, expected_tag):
```

Comment:
nit, redundant "tag" word in test methods, this and related could be renamed to:
```suggestion
    def test_valid_prefixes_and_suffixes(self, docker_context_dir, expected_tag):
```